src/components/pages/Chat/AddRoomModal/AddRoomItem.tsx

```diff
@@ -191,22 +215,36 @@ function UnstyledAddRoomItem(props: Props) {
                             </CloseButton>
                         </ModalHeader>
                         <div>
-                            <Subheading>Name</Subheading>
-                            <input
-                                value={roomName}
-                                onChange={handleChangeRoomName}
-                            />
-                            <Subtitle>
-                                Room name is generated randomly but you can
-                                provide your own, as long as it's unique in your
-                                account. The room name will be publicly visible.
-                                <br />
-                                You may use alphanumeric characters, as well as
-                                dashes (-) and underscores (_) for room names.{' '}
-                            </Subtitle>
-                            <CreateButton onClick={initCreateRoom}>
-                                Create
-                            </CreateButton>
+                            <form>
```

Comment:
Why `form`?

If we really want to do this let's refactor the submit button, too. Move the call to `initCreateRoom` into `<form onSubmit={initCreateRoom}>` and change `CreateButton`'s type to `submit`.

This way you can also submit the form using `Return` key on your keyboard while typing the name. UX!